spec/jobs/process_photo_job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.